glide-core/src/client/value_conversion.rs

```diff
@@ -607,6 +608,50 @@ pub(crate) fn convert_to_expected_type(
             }
             _ => Err((ErrorKind::TypeError, "Response couldn't be converted").into()),
         },
+        // Used by XAUTOCLAIM. The command returns a list of length 2 if the Redis version is less than 7.0.0 or a list
+        // of length 3 otherwise. It has the following response format:
+        /* Redis version < 7.0.0 example:
```

Comment:
Is it the same response with RESP3 too?

Reply:
as far as I can tell, resp2 and resp3 have the same response. The response only differs if you are using redis version < 7.0.0 compared to redis version >= 7.0.0